Vic2ToHoI4/Source/HOI4World/HoI4Localisation.cpp

```diff
@@ -13,6 +14,56 @@
 #include <fstream>
 
 
+namespace
+{
+std::optional<Vic2::LanguageToLocalisationMap> getConfigurableDominionNames(
```

Comment:
This could be tested

Reply:
It's a purely internal function (the anonymous namespace trick ensures that), so the only testing should be in how it affects the functions that call it. Which sadly are not tested.